modules/gdnative/godot/godot_rid.h

```diff
@@ -27,31 +27,36 @@
 /* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
-#ifndef GODOT_RID_H
-#define GODOT_RID_H
+#ifndef GODOT__RID_H
+#define GODOT__RID_H
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 #include <stdint.h>
+#include <wchar.h>
 
-#ifndef GODOT_CORE_API_GODOT_RID_TYPE_DEFINED
+#ifndef GODOT_CORE_API_GODOT__RID_TYPE_DEFINED
```

Comment:
Why the double underscores? The underscore was added in Godot because of a name collision, here it's not the case.